test/CodeGenerator/InjectorGeneratorTest.php

```diff
@@ -74,11 +74,10 @@ public function testGeneratorLogsDebugForEachClass()
     {
         $config = new Config();
         $resolver = new DependencyResolver(new RuntimeDefinition(), $config);
-        $generator = new InjectorGenerator($config, $resolver);
         $logger = $this->prophesize(LoggerInterface::class);
 
+        $generator = new InjectorGenerator($config, $resolver, null, $logger->reveal());
```

Comment:
Did you also change the constructor, or was it already accepting an optional logger?

Reply:
This was the test, only. Just commited the code changes ;-)